pkg/kubelet/cm/cpumanager/policy_options_test.go

```diff
@@ -109,53 +109,75 @@ func TestPolicyOptionsAvailable(t *testing.T) {
 
 func TestValidateStaticPolicyOptions(t *testing.T) {
 	testCases := []struct {
-		description   string
-		policyOption  map[string]string
-		topology      *topology.CPUTopology
-		topoMgrPolicy string
-		expectedErr   bool
+		description         string
+		policyOption        map[string]string
+		topology            *topology.CPUTopology
+		topoMgrPolicy       string
+		expectedErr         bool
+		expectedValidateErr bool
 	}{
 		{
-			description:   "Align by socket not enabled",
-			policyOption:  map[string]string{FullPCPUsOnlyOption: "true"},
-			topology:      topoDualSocketMultiNumaPerSocketHT,
-			topoMgrPolicy: topologymanager.PolicySingleNumaNode,
-			expectedErr:   false,
+			description:         "Align by socket not enabled",
+			policyOption:        map[string]string{FullPCPUsOnlyOption: "true"},
+			topology:            topoDualSocketMultiNumaPerSocketHT,
+			topoMgrPolicy:       topologymanager.PolicySingleNumaNode,
+			expectedValidateErr: false,
+		},
+		{
+			description:         "Distribute CPUs accros numa nodes",
+			policyOption:        map[string]string{DistributeCPUsAcrossNUMAOption: "true"},
+			topology:            topoDualSocketMultiNumaPerSocketHT,
+			topoMgrPolicy:       topologymanager.PreferClosestNUMANodes,
+			expectedValidateErr: false,
 		},
 		{
-			description:   "Align by socket enabled with topology manager single numa node",
-			policyOption:  map[string]string{AlignBySocketOption: "true"},
+			description:   "Distribute CPUs accros numa nodes with invalid policy",
+			policyOption:  map[string]string{DistributeCPUsAcrossNUMAOption: "invalid"},
 			topology:      topoDualSocketMultiNumaPerSocketHT,
-			topoMgrPolicy: topologymanager.PolicySingleNumaNode,
+			topoMgrPolicy: topologymanager.PreferClosestNUMANodes,
 			expectedErr:   true,
```

Comment:
`expectedValidateErr` is missing here. You should either put it everywhere or only use it when it's value is `true`.

Reply:
My bad, sorry. Resolved in 832cb5ad9e3